tests/Unit/Controllers/AdminControllerTest.php

```
@@ -2541,7 +2547,11 @@ public function testBillingWithMockStripeHttpClientCoversAllLines(): void
             $this->stmt(false),                      // Integration (xero) - not active
         ]);
 
-        $this->ctrl()->billing();
+        $cfg = $this->config;
+        $cfg['stripe']['dashboard_live_fetch'] = true;
+
+        $ctrl = new AdminController($this->makeGetRequest(), $this->response, $this->auth, $this->db, $cfg, $this->probeFactory($cfg));
+        $ctrl->billing();
 
         // Restore default HTTP client
         \Stripe\ApiRequestor::setHttpClient(null);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Stripe\ApiRequestor' in method 'testBillingWithMockStripeHttpClientCoversAllLines'. (undefined)

(StaticAccess)